app/vmauth/README.md

```diff
@@ -341,6 +341,8 @@ See the docs at https://docs.victoriametrics.com/vmauth.html .
 
   -auth.config string
      Path to auth config. It can point either to local file or to http url. See https://docs.victoriametrics.com/vmauth.html for details on the format of this auth config
+  -brokenBackendBackoff duration
```

Comment:
I'm pretty happy with the PR overall, except flag name. It seems not intuitive to me, because on the moment of reviewing, I didn't know about `backend` or `broken` entities. Config doesn't seem to have such references as well.
Shall we try another name, like well known `fail_timeout` (camel-cased) from nginx? 
Unless, @SunKyu @zekker6 and @Amper are all agree that `brokenBackendBackoff` is the right name to use.

Reply:
I agree `fail_timeout` , if @zekker6 @Amper don't mind, i will try `fail_timeout`

Reply:
Sure. Sounds good to me

Reply:
I changed to failTimeout